AWS/terraform/README.md

```diff
@@ -16,22 +16,22 @@ Once all prerequisites are installed you confirmed that terraform successfully a
 
 1. Review the `variables.tf` file. This file contains all variables for the deployment. Most of the variables have default values assigned, but can be changed in the `.tf` file or overridden by using the `-var` flag with the `terraform apply` command. You will be prompted for any variable that does not have a default after running the `terraform apply` command.
 2. Run `terrafom apply` in your console from the directory that that holds the `main.tf` and `variables.tf` file and provide any variables you are prompted for.
-3. Review the deployment plan. If the terraform script and the provided variables validated successfully the deployment plan will be output in the consoled for you to review. Additionally you will be prompted wether you want to go ahead with the deployment. If everything looks ok, go ahead with `yes`. The deployment will now provision and configure all necessary AWS services and start up FME Server. This will take about 10 - 20 minutes.
+3. Review the deployment plan. If the terraform script and the provided variables validated successfully the deployment plan will be output in the consoled for you to review. Additionally you will be prompted wether you want to go ahead with the deployment. If everything looks ok, go ahead with `yes`. The deployment will now provision and configure all necessary AWS services and start up FME Flow. This will take about 10 - 20 minutes.
```

Comment:
Grammar:  If the terraform script and the provided variables _**are**_ validated successfully the deployment plan will be _**outputted**_ in the consoled for you to review.

Reply:
Thanks! In this case I like `output` better than `outputted`, but I have made the other changes, and also found we said "consoled" here instead of "console".